### PR TITLE
Update the background menu color when no theme is set.

### PR DESCRIPTION
**Pull request by CSilivestru**
Currently displays white on white when no colour theme is provided to the the top bar. The defaults provided didn't seem correct so I'm updating them :).

### WHY are these changes introduced?

Addresses issue https://github.com/Shopify/destinations/issues/760.

### WHAT is this pull request doing?
Before:
image

After:
image

image

## 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}

```

</details>

### 🎩 checklist

* [x] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
* [x] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
* ~~[ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)~~N/A
* [x] Updated the component's `README.md` with documentation changes
* ~~[ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide~~
* [x] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit

### CODE REVIEW

**Comment by CSilivestru** [comment]
I made a change to include a var within a var to have polaris attempt to use the background lighter theme override if provided.

The reason this didn't work in earlier trials is because the css variable fallback is treated as plain text and doesn't invoke any functions -- see [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#Custom_property_fallback_values).

Because of this, I had to remove the hover variable and place it directly in as the fallback. I'm more than happy to discuss alternatives to this!

File: src/components/TopBar/README.md [hunk 1]
```diff
@@ -370,6 +372,7 @@ function TopBarExample() {
       onChange={handleSearchChange}
       value={searchValue}
       placeholder="Search"
+      showFocusBorder
```
Comment (alex-page) [inline src/components/TopBar/README.md:375]: To make it easier for the plus team, I updated the example to use their white theme.

File: src/components/TopBar/components/Menu/Menu.scss [hunk 2]
```diff
@@ -2,11 +2,7 @@
 
 $activator-variables: (
   min-width: rem(36px),
-  border-left: 1px solid rgba(color('black'), 0.42),
   transition: background-color 0.1s,
-  focus-background-color: rgba(color('white'), 0.16),
-  hover-background-color: rgba(color('white'), 0.08),
-  active-background-color: rgba(color('black'), 0.28),
```
Comment (alex-page) [inline src/components/TopBar/components/Menu/Menu.scss:9]: These values are used once and blocked multiple theme overrides.

**Comment by alex-page** [comment]
@AlexBlaise you should be able to TopHat this when the tests pass there will be a Chroma check that will have a link to an example.

**Pull request merged**